app/views/plan/index.scala

```diff
@@ -290,14 +290,12 @@ object index {
         ),
         dt(otherMethods()),
         dd(
-          "Lichess is ",
-          a(href := "https://causes.benevity.org/causes/250-5789375887401_bf01")("registered with Benevity"),
-          ".",
+          lichessIs(a(href := "https://causes.benevity.org/causes/250-5789375887401_bf01")(registeredWithBenevity())),
```

Comment:
this is called "translation lego" and it's bound to fail with a number of languages.

You have to put the link either on the entire sentence or, and it's better imo, on the single untranslated word "Benefity"